packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -5,6 +5,8 @@
  * and https://github.com/palantir/blueprint/blob/master/PATENTS
  */
 
+const POPOVER_HANDLE_WIDTH = 25;
```

Comment:
I'd love to rename this `ASSUMED_ POPOVER_HANDLE_WIDTH` to make it abundantly clear that this is dependent on CSS styling. Would also be helpful to add a comment in the CSS mentioning that this value here needs to be kept in sync with any style changes (and maybe add a comment here too?)